qutip/tests/solver/test_sesolve.py

```diff
@@ -79,6 +80,29 @@ def test_sesolve(self, H, analytical, unitary_op):
         np.testing.assert_allclose(sy, sy_analytic, atol=tol)
         np.testing.assert_allclose(sz, sz_analytic, atol=tol)
 
+    @pytest.mark.parametrize(['state_type'], [
+        pytest.param("ket", id="ket"),
+        pytest.param("unitary", id="unitary"),
+    ])
+    def test_sesolve_normalization(self, state_type):
+        # non-hermitean H causes state to evolve non-unitarily
+        H = qutip.Qobj([[1, -0.1j], [-0.1j, 1]])
+        psi0 = qutip.basis(2, 0)
+        options = SolverOptions(normalize_output=True, progress_bar=None)
+
+        if state_type == "ket":
+            output = sesolve(H, psi0, self.tlist, e_ops=[], options=options)
+            norms = [state.norm() for state in output.states]
+            np.testing.assert_allclose(
+                norms, [1.0 for _ in self.tlist], atol=5e-3,
```

Comment:
```suggestion
                norms, [1.0 for _ in self.tlist], atol=1e-15,
```
This should be very precise.